### PR TITLE
deps: bump shared-deps version and add back certain test deps

### PR DESCRIPTION
**Pull request by stephaniewang526**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -122,6 +122,26 @@
...
+      <dependency>
+        <groupId>com.google.truth</groupId>
+        <artifactId>truth</artifactId>
+        <version>1.0.1</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>grpc-google-iam-v1</artifactId>
+        <version>0.13.0</version>
```
Comment (chingor13) [inline pom.xml:142]: This one also seems like it should be managed by the shared-deps

Comment (stephaniewang526) [reply pom.xml:142]: ok - adding this to shared-deps https://github.com/googleapis/java-shared-dependencies/pull/55

Comment (stephaniewang526) [reply pom.xml]: done

**Comment by frankyn** [comment]
Thanks @stephaniewang526!

**Pull request merged**